observation-service/src/main/java/gov/cdc/etldatapipeline/observation/util/ProcessObservationDataUtil.java

```diff
@@ -323,10 +323,12 @@ private void transformObservationCoded(String observationCoded) {
         try {
             JsonNode observationCodedJsonArray = parseJsonArray(observationCoded);
 
+            ObservationCodedKey codedKey = new ObservationCodedKey();
             for (JsonNode jsonNode : observationCodedJsonArray) {
                 ObservationCoded coded = objectMapper.treeToValue(jsonNode, ObservationCoded.class);
-                observationKey.setObservationUid(coded.getObservationUid());
-                sendToKafka(observationKey, coded, codedTopicName, coded.getObservationUid(), "Observation Coded data (uid={}) sent to {}");
+                codedKey.setObservationUid(coded.getObservationUid());
```

Comment:
Are there any sink issues when we add a new key value? The table is set to expecting a single id.

Reply:
Hopefully not. This is a kafka key for the respective topic and the actual payload will still have the same schema as before.
Never the less let's test it out in DTS1.

Reply:
Agreed. We usually have some issues when we change the schema of the key.